client/src/app/containers/Game/PlayingCard.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+const PlayingCard = props => {
+  if (!props.name) {
+    return null;
+  }
```

Comment:
We can use `PropTypes` to [validate props](https://reactjs.org/docs/typechecking-with-proptypes.html)